src/app.ts

```diff
@@ -1017,6 +1017,20 @@ export const initializeAppModules = () => {
       }
     });
 
+    addModule({
+      name: "shareToMeetingStage",
+      initializedRequired: true,
+      action: function () {
+        let appContext = {
+          appSharingUrl: "https://teams-test-tab.azurewebsites.net",

```

Comment:
In our tests, this app gets hosted at https://statics.teams.microsoft.com so it might be better to use window.location.href when building this URL.